libs/dexie-cloud-common/src/tson/types/Blob.ts

```
@@ -0,0 +1,30 @@
+      v:
+        blob instanceof FakeBlob
+          ? b64encode(blob.buf)
+          : b64encode(string2ArrayBuffer(readBlobSync(blob))),
+      type: blob.type,
+    }),
+    revive: ({ type, v }: { type: string; v: string }) => {
+      const ab = b64decode(v);
+      const buf = ab.buffer.byteLength === ab.byteLength
+        ? (ab.buffer as ArrayBuffer)
+        : (ab.buffer as ArrayBuffer).slice(ab.byteOffset, ab.byteOffset + ab.byteLength);
+      return typeof Blob !== undefined
```

Biome (2.3.13)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)